pallets/utxo/src/lib.rs

```diff
@@ -179,6 +189,15 @@ pub mod pallet {
                 destination: Destination::CreatePP(code, data),
             }
         }
+
+        /// Create a new output to create a smart contract.
+        pub fn new_call_pp(value: Value, dest_account: AccountId, input: Vec<u8>) -> Self {
```

Comment:
This is a misleading comment as OP_CALL does not create a new smart contract

Reply:
Good catch, this is a copy/paste error, will fix